sdk/eventhub/Azure.Messaging.EventHubs/tests/EventHubConsumer/EventHubConsumerLiveTests.cs

```diff
@@ -474,6 +540,308 @@ await using (var consumer = client.CreateConsumer(EventHubConsumer.DefaultConsum
+                    await using (var producer = client.CreateProducer(new EventHubProducerOptions { PartitionId = partition }))
+                    {
+                        // Sending some events beforehand so the partition has some information.
+
+                        for (int i = 0; i < 10; i++)
+                        {
+                            await producer.SendAsync(new EventData(new byte[1]));
+                        }
+
+                        // Receive and validate the events; because there is some non-determinism in the messaging flow, the
+                        // sent events may not be immediately available.  Allow for a small number of attempts to receive, in order
+                        // to account for availability delays.
+
+                        var receivedEvents = new List<EventData>();
+                        var index = 0;
+
+                        while ((receivedEvents.Count < 10) && (++index < ReceiveRetryLimit))
+                        {
+                            receivedEvents.AddRange(await consumer.ReceiveAsync(20, TimeSpan.FromMilliseconds(25)));
+                        }
+
+                        Assert.That(receivedEvents.Count, Is.EqualTo(10), "The number of received events should be 10.");
+                    }
+                }
+            }
+        }
+
+        /// <summary>
+        ///   Verifies that the <see cref="EventHubConsumer" /> is able to
+        ///   connect to the Event Hubs service and perform operations.
+        /// </summary>
+        ///
+        [Test]
+        [TestCase(true)]
+        [TestCase(false)]
+        [Ignore("isInclusive parameter is not supported yet")]
+        public async Task ConsumerCanReceiveFromOffset(bool isInclusive)
+        {
+            await using (var scope = await EventHubScope.CreateAsync(1))
+            {
+                var connectionString = TestEnvironment.BuildConnectionStringForEventHub(scope.EventHubName);
+
+                await using (var client = new EventHubClient(connectionString))
+                {
+                    var partition = (await client.GetPartitionIdsAsync()).First();
+
+                    await using (var producer = client.CreateProducer(new EventHubProducerOptions { PartitionId = partition }))
+                    {
+                        // Sending some events beforehand so the partition has some information.
+
+                        for (var i = 0; i < 10; i++)
+                        {
+                            await producer.SendAsync(new EventData(new byte[1]));
+                        }
+
+                        // Store last enqueued offset.
+
+                        var offset = int.Parse((await client.GetPartitionPropertiesAsync(partition)).LastEnqueuedOffset);
+
+                        // We must remember to include the 'isInclusive' flag when creating the consumer.
```

Comment:
This comment does not appear to be accurate.  :)

Reply:
I left it as a reminder for when we enable the `isInclusive` flag. We can't include it now.

Reply:
Given that we won't have an `isInclusive` flag for offsets, simply removed the comment.